oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyState.java

```diff
@@ -64,7 +65,8 @@ final class DocumentPropertyState implements PropertyState {
     private final byte[] compressedValue;
     private final Compression compression;
 
-    private static final int DEFAULT_COMPRESSION_THRESHOLD = Integer.getInteger("oak.mongo.compressionThreshold", -1);
+    private static final int DEFAULT_COMPRESSION_THRESHOLD = SystemPropertySupplier
+            .create("oak.documentMk.stringCompressionThreshold ", -1).loggingTo(LOG).get();
```

Comment:
```suggestion
            .create("oak.documentMK.stringCompressionThreshold ", -1).loggingTo(LOG).get();
```

oups, my bad - looks like we use `MK` rather than `Mk` ...